src/components/controls/FilterListPanel.jsx

```diff
@@ -10,8 +9,6 @@ import {
 /** recursively get an array of node keys to toggle */
 function getFiltersToToggle(filter, activeFilters) {
   const [key, children] = filter;
```

Comment:
esbuild correctly identified this case as always being falsy - removed the assertion to match the current behavior